SeaPublicWebsite.BusinessLogic/Models/Enums/AccessibleLoft.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GovUkDesignSystem.Attributes;
+
+namespace SeaPublicWebsite.BusinessLogic.Models.Enums
+{
+    public enum AccessibleLoft
+    {
+        [GovUkRadioCheckboxLabelText(Text = "I'm not sure")]
```

Comment:
There isn't a "Don't Know" option in the design?

Reply:
See below about EPC hint